MTDome re-Home Tickets/sitcom 1438

PaulinaLSST opened the pull request:
Dear colleague, 

This is a simple and temporary document, but it needs to be reviewed.
This ticket contains the steps for re-homing the Main Dome.

Cheers,

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+Error Diagnosis
+===============
+
+* While the dome is not positioned where it should be, the images might be vignetted by the dome. In more extreme cases, no images show at all if the cameras are completely covered by the dome.
```
Grammatically incorrect. Suggest --- > "If the MTDome is not positioned where it should be, the images from LSSTCam/ComCam/Startracker may get vignetted by the MTDome. In extreme case, the images will show nothing if the LSSTCam/ComCam/StarTracker cameras are covered by the MTDome (i.e if the MTDome slit does not coincide with the cameras' pointing )"

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+===============
+
+* While the dome is not positioned where it should be, the images might be vignetted by the dome. In more extreme cases, no images show at all if the cameras are completely covered by the dome.
+* The azimuth position reported by the dome is not the same as the one seen in the camera.
```
Needs clarification -- 
1. where do you check the MTDome azimuth position? 
2. How do you check the MTDome azimuth position?

Needs a picture from the Unifi Camera to show how it actually looks through webcam.

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+Procedure Steps
+===============
+
+1. Move the dome to the azimuth of 328 deg, the position can be verified through the camera systems UniFi at `Cam-04`_ (via VPN). This is the *zero position* **or** *park* position for the dome.
```
convert  *zero position* **or** *park*  to Boldface

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+
+* For example, you notice the dome is reporting 10 degree in azimuth, but the camera marks 25.2 degrees (the encoder is 15.2 degrees behind).
+
+To send the dome to home at 328 degrees, you need to command the dome to move to *(328 - 15.2) = 312.8* degrees.
```
1. Convert this *(328 - 15.2) = 312.8*  to boldface
2. Move this before point 2

Kelks19 replied:
Or remove the bullet on point 1 and keep it as an opening summary statement.

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+        position: 312.8
+..
+
+3. Once the dome is in the *zero position*, you need to stop the dome and engage the breaks.
```
Convert *zero position* to boldface

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+..
+
+
+4. Set the dome *zero azimuth* position by running :file:`run_command.py` script with the following configuration:
```
convert *zero azimuth* to boldface

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+
+Post-Condition
+==============
+* MTDome encoder is showing the right value and operations can continue.
```
.........showing the right value in the Unifi camera ...

Kelks19 commented on Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst:
```diff
@@ -0,0 +1,93 @@
+
+1. Move the dome to the azimuth of 328 deg, the position can be verified through the camera systems UniFi at `Cam-04`_ (via VPN). This is the *zero position* **or** *park* position for the dome.
+   
+2. Note that the readings reported by the dome in LOVE or Chronograf will show different numbers, so you need to calculate the value that the unaligned encoders would read at the park position of 328 degrees and send the dome to that angle.
```
1. Which dashboard in LOVE and /or Chronograf? Need to specify  
2.  "LOVE" and "Chronograf" should be in italics

Kelks19 reviewed (changes requested):
The document needs a bit of work with respect to content and formatting. 

For the content-- 
1. Missing information on how do we check the recorded MTDome position (e.g how do we check this in LOVE etc)

For formatting -- 
Most of this is already highlighted [here](https://rubinobs.atlassian.net/wiki/spaces/OOD/pages/39698537/Checklist+for+reST+documentation)  but I summarise as -
1. software names/paths etc should be in italics 
2. any fact you might want to highlight should be in Boldface 
3. dome should always be referred to as MTDome
4. guilabel:'' for Buttonsin EUI /LOVE etc 
5. monospace to denote parameter values

Kelks19 reviewed (changes requested):
I have already left me comments which are not yet addressed.

PaulinaLSST commented:
Hi 
I have addressed all your comments in this document. Could you please re-review it to verify that the comments were well addressed?

Thanks :)

PaulinaLSST commented:
Hi @Kelks19 
I have addressed all your comments in this document. Could you please re-review it to verify that the comments were well addressed?

Thanks :)

PaulinaLSST commented:
Document version at
https://obs-ops.lsst.io/v/SITCOM-1438/Simonyi/Troubleshooting/MTCS/MTDome/MTDome-re-home.html

PaulinaLSST commented:
KK and Ioana.
Could you please review this document that has been pending for upload to the Obs-Op-Doc since last year? @isotuela @Kelks19

PaulinaLSST commented:
Opening new PR.
